low-level/framework/utils/sysfs_interface.py

```diff
@@ -58,6 +60,13 @@ def initialize(self):
         except OSError as os_error:
             return os_error.errno
 
+    def get_sysfs_base_path(self):
+        """Returns the  sysfs base path 
+        Ex: /sys/ """
+
+        sysfs_base_path = Conf.get(SSPL_CONF, "SYSTEM_INFORMATION>sysfs_base_path")
```

Comment:
If conf get fails, what  default value will be returned ?

Reply:
Added the default value ('/sys/') to the api.